prusti-contracts/src/lib.rs

```diff
@@ -78,8 +81,10 @@ mod private {
     /// A macro for defining a predicate using prusti expression syntax instead
     /// of just Rust expressions.
     pub use prusti_contracts_internal::predicate;
-}
 
+    /// Macro for creating type [models](prusti_contracts_internal::model).
```

Comment:
```suggestion
    /// Macro for creating type models.
```
(Do re-export docs do anything in the IDE?)